scout/models/variant/variant.py

```diff
@@ -3,12 +3,9 @@
 variant = dict(
     # document_id is a md5 string created by institute_genelist_caseid_variantid:
     _id=str,  # required, same as document_id
-    document_id=str,  # required
-    # variant_id is a md5 string created by chrom_pos_ref_alt (simple_id)
-    variant_id=str,  # required
-    # display name is variant_id (no md5)
-    display_name=str,  # required
-    # chrom_pos_ref_alt
+    document_id=str,  # required. Same as _id
+    variant_id=str,  # required. A md5 string created by chrom_pos_ref_alt_[clinical/research]
```

Comment:
👍

Reply:
(you could call that `[c/r]` variant_type as well; defined below)

Reply:
Done!